code/website/index.html

```diff
@@ -1,22 +1,47 @@
 <h2>Drasil - Generate All the Things!</h2>
+<p id="center"><a href="https://github.com/JacquesCarette/Drasil">Link to GitHub repository</a></p>
+<hr>
+<br />
+<p>
+  This webpage is designed to contain the most up to date
+  <a href="#caseStudies">case study artifacts</a>
+  and <a href="#packages">package dependency graphs</a> from the Drasil repository.
+  The case study artifacts include the Software Requirements Specification (SRS) for the case study,
+  which specifies what the program sets out to achieve.
+  The package dependency graphs shows the hierarchy of modules within each package.
+  The footer of this page contains the continuous integration build of the project,
+  as well as the commit number that the build and artifacts are based off of.
+</p>
+<br />
+<a name="caseStudies"></a>
```

Comment:
`<a name>` is [not valid in HTML5](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). And an empty link with an "id" is incorrect here. The id `caseStudies` should be given to the `<h3>` on the following line.